rfc/graphql-schema/additionalTypeDefs/byPage/userManagement.ts

```diff
@@ -0,0 +1,108 @@
+export default /* GraphQL */ `
+  input CustomerFiltersInput {
+    search: String
+    companyRoleId: ID
+  }
+
+  type Role {
+    id: ID!
+    name: String!
+  }
+
+  extend type Customer {
+    role: Role
+  }
+
+  type CustomerEdge {
+    node: Customer!
+    cursor: String!
+  }
+
+  type CustomerConnection {
+    edges: [CustomerEdge!]
+    pageInfo: PageInfo!
+  }
+
+  type Company implements Node {
+    id: ID!
+    Customers(
```

Comment:
I know we went for Customers to follow suit with the current schema, I wonder however if we should call them Users to follow b2b terminology 🤔

Reply:
also, why is it camel cased?

Reply:
the casing was just a mistake

a `company.users` query that brings back `customer[]` is a bit of an odd mix 🤔

Reply:
I've added a `openQuestions.md` file to collect these

we may gain extra clarity as we add more to the draft schema
even if not, we can collect them together to work through in as a group